wordpress-mobile/WordPress-iOS
# Merge Release notes into `develop`

**To test:**
Ensure tests pass
Ensure new release notes are present
Ensure version numbers are going up

Update release notes:
